tests/events/client/instrumentation/test_task_run_state_change_events.py

```diff
@@ -41,163 +62,357 @@ def happy_path():
 
     pending, running, completed = events
 
-    assert pending.event == "prefect.task-run.Pending"
-    assert pending.id == task_run_states[0].id
-    assert pending.occurred == task_run_states[0].timestamp
-    assert pending.resource == Resource(
-        {
-            "prefect.resource.id": f"prefect.task-run.{task_run.id}",
-            "prefect.resource.name": task_run.name,
-            "prefect.state-message": "",
-            "prefect.state-type": "PENDING",
-            "prefect.state-name": "Pending",
-            "prefect.state-timestamp": task_run_states[0].timestamp.isoformat(),
-            "prefect.orchestration": "server",
+    if enable_client_side_task_run_orchestration:
```

Comment:
theres just enough differences that I opted to have one conditional here instead of sporadic conditionals throughout the comparisons. this also makes it easier in the future for removal